File: java/src/view/MenuView.java
package view;

import java.util.Scanner;

public class MenuView {

    private Scanner entrada;

    public MenuView() {

        entrada = new Scanner(System.in);
    }

    public int mostrarMenu() {

        System.out.println("\n===== RPG MANAGER =====");
        System.out.println("1. Crear personaje");
        System.out.println("2. Listar personajes");
        System.out.println("3. Buscar personaje");
        System.out.println("4. Actualizar nivel");
        System.out.println("5. Eliminar personaje");
        System.out.println("6. Salir");

        System.out.print("Seleccione opcion: ");

        return entrada.nextInt();
    }

    public String pedirNombre() {

        entrada.nextLine();

        System.out.print("Ingrese nombre: ");

        return entrada.nextLine();
    }

    public int pedirNivel() {

        System.out.print("Ingrese nivel: ");

        return entrada.nextInt();
    }

    public int pedirId() {

        System.out.print("Ingrese ID: ");

        return entrada.nextInt();
    }

    public int pedirClase() {

        System.out.println("1. Guerrero");
        System.out.println("2. Mago");
        System.out.println("3. Arquero");

        System.out.print("Seleccione clase: ");

        return entrada.nextInt();
    }

    public String pedirNombreBusqueda() {

        entrada.nextLine();

        System.out.print("Ingrese nombre a buscar: ");

        return entrada.nextLine();
    }

    public void mostrarMensaje(String mensaje) {

        System.out.println(mensaje);
    }
}

File: python/view/menu_view.py
class menuview:

    # Mostrar menú
    def mostrar_menu(self):

        print("===== RPG MANAGER =====")
        print("1. Crear personaje")
        print("2. Listar personajes")
        print("3. Actualizar nivel")
        print("4. Eliminar personaje")
        print("5. Salir")

        # retorna opción
        return int(input("Seleccione opcion: "))

    # Pedir ID
    def pedir_id(self):

        return int(input("Ingrese ID: "))

    # Pedir nombre
    def pedir_nombre(self):

        return input("Ingrese nombre: ")

    # Pedir nivel
    def pedir_nivel(self):

        return int(input("Ingrese nivel: "))

    # Pedir clase
    def pedir_clase(self):

        return int(
            input("Seleccione clase: ")
        )

    # Mostrar mensaje
    def mostrar_mensaje(self, mensaje):

        print(mensaje)